ciao-controller/client.go

```diff
@@ -114,26 +109,48 @@ func (client *ssntpClient) releaseResources(instanceID string) error {
 	return nil
 }
 
-func (client *ssntpClient) removeInstance(instanceID string) {
-	err := client.releaseResources(instanceID)
+func (client *ssntpClient) removeInstance(i *types.Instance) {
```

Comment:
@rbradford I reviewed the change again and think that this change was made in an earlier revision of the patch, and is no longer relevant. I'll revert it back to the original implementation.